manifests/fine-tuning-6b/core-modules.yaml

```diff
@@ -45,7 +45,7 @@ parameters:
             usage: core
         - eks_ng_name: ng-gpu
           eks_node_quantity: 6
-          eks_node_max_quantity: 15
+          eks_node_max_quantity: 10
```

Comment:
Is this change intentional?

Reply:
Yes, thank you for checking though. 15 instances is an overkill for this training so I thought might as well save customers some $ :)